scripts/doc-checker.py

```diff
@@ -33,22 +33,23 @@ def validate_object(import_path: str) -> int:
 
     Returns
     -------
-    exit_status : int
+    is_failed : bool
```

Comment:
Using a double-negative thing is usually a bad practice... I suggest changing the meaning to "`True` when object is valid", which seems more logical to me.

Here and in other places, too.

Reply:
updated